doc/source/whatsnew/v2.1.0.rst

```diff
@@ -682,6 +682,7 @@ Other
 - Bug in :class:`DataFrame` and :class:`Series` raising for data of complex dtype when ``NaN`` values are present (:issue:`53627`)
 - Bug in :class:`DatetimeIndex` where ``repr`` of index passed with time does not print time is midnight and non-day based freq(:issue:`53470`)
 - Bug in :class:`FloatingArray.__contains__` with ``NaN`` item incorrectly returning ``False`` when ``NaN`` values are present (:issue:`52840`)
+- Bug in :func:`api.interchange.from_dataframe` was raising during interchanging from non-pandas tz-aware data containing ``NaN`` values (:issue:`54287`)
```

Comment:
can we write
```
non-pandas tz-aware data containing null values
```
?
technically `nan` is different, and specifically refers to invalid mathematical operations like 0/0

Reply:
oops, my mistake. I corrected that.